client/components/domains/connect-domain-step/constants.ts

```diff
@@ -46,9 +46,16 @@ export const defaultDomainSetupInfo = {
 	},
 } as const;
 
+export const domainLockStatusType = {
+	LOCKED: 'locked',
+	UNLOCKED: 'unlocked',
+	UNKNOWN: 'unknown',
+} as const;
+
 export const stepsHeadingSuggested = __( 'Suggested setup' );
 export const stepsHeadingAdvanced = __( 'Advanced setup' );
 export const stepsHeadingOwnershipVerification = __( 'Verify domain ownership' );
+export const stepsHeadingTransfer = __( 'Initial setup' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Finishing initial setup' )` *ES Score: 7*